### ISSUE TITLE
ng-otp-input Accessibility Issue

### ISSUE THREAD

**Issue by RishabhNagpal** [code-farmz/ng-otp-input#41]
I'm submitting a ... (check one with "x")

[x] bug report => see 'Providing a Reproducible Scenario'

Requesting Community Support

Current behaviour
I am facing an accessibility issue in using ng-otp-input library. When I run wave tool to check for accessibility errors, I get an error that a form control does not have a label for each input box.

Providing a Reproducible Scenario
Here is the link showing the scenario. Please install the wave extension for chrome and run it on the page.
https://github-tgkvfi.stackblitz.io/

Expected behaviour
Each input box should have a label.

Environment:
Windows 10, IDE visual studio code

ng-otp-input version : 1.7.1

Browser:
Chrome

Language:
Typescript

**Comment by code-farmz** [comment]
@RishabhNagpal  this is just an input box component and doesn't need to have labels with each input as per functionality

**Comment by arpitmehrotra** [comment]
Hi @code-farmz, 
While doing an a11y check on the OTP input box, there should be some way to explicitly set the aria-label attribute to each input right for screen reader to spell it out correctly (currently it reads it as "input blank")
Can you let us know if there is a workaround for that?

**Comment by code-farmz** [comment]
@arpitmehrotra get elements by the class name `otp-input` or pass custom input class if needed and then set the area-label attribute 

```
var eles=document.getElementsByClassName('otp-input')
for (let ele of eles) {
    ele.setAttribute('aria-label', 'slide-dot')
}
```
